spec/models/pglogical_subscription_spec.rb

```diff
@@ -241,6 +273,9 @@
 
       expect { sub.save! }.to raise_error(RuntimeError, "Some Error")
     end
+
+    it "raises when the schemas don't match" do
```

Comment:
match might be ok, but it's not clear what it matches against.  I prefer something that indicates validity because we're not checking everything in the schemas, only what validation checks we have against the master/good schema.